en/core-api/bank-transfer.md

```diff
@@ -274,44 +464,91 @@ You will get the `bill_key` and `bill_code` attribute which can be performed thi
+
+</article>
+</details>
+
+?>***Note:*** The `permata_va_number` attribute for the transaction is received.
+
 <!-- tabs:end -->
 
-## 2. Display Virtual Account Number and Expiry Time
+#### 2. Displaying Virtual Account Number and Expiry Time
 
 To display the virtual account number, use the value of `va_number` retrieved from API response.
 
 ?> Read [here to simulate/test success payment](/en/technical-reference/sandbox-test.md#bank-transfer).
 
-By default the expiry time for Bank Transfer / VA is **24 hours**, follow this link if you wanna customize the expiry time:
+By default the expiry time for Bank Transfer / VA is **24 hours**. Follow the link given below to customize the expiry time:
 <div class="my-card">
 
 #### [Set Custom Expiry](/en/core-api/advanced-features.md#custom-transaction-expiry)
 </div>
 
-## 3. Handling Post-Transaction
+#### 3. Handling Transaction Notification
 
-Other than customer being redirected, when the status of payment is updated/changed (i.e: payment has been successfully received), Midtrans will send **HTTP Notification** (or webhook) to your server's `Notification Url` (specified on Midtrans Dashboard, under menu **Settings > Configuration `Notification URL`**). Follow this link for more details:
+When the transaction status changes, Midtrans notifies you at the redirect URL and sends HTTP notification to the merchant backend. This ensures that you are updated of the transaction status securely.
+
+HTTP POST request with JSON body will be sent to your *Payment Notification URL* configured on dashboard.
+
+<details>
+<summary><b>Configuring Payment Notification URL</b></summary>
+<article>
+To configure the Payment Notification URL, follow the steps given below.
+
+1. Login to your MAP account.
+
+2. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed.
+
+3. Enter **Payment Notification URL**.
+
+4. Click **Update**.
+   A confirmation message is displayed.
+
+   ![Core API](./../../asset/image/coreapi/core-api-payment-notification-1.png)
+
+   The *Payment Notification URL* is configured.
+
+   </article>
+   </details>
 
-<div class="my-card">
 
 #### [Handling Webhook HTTP Notification](/en/after-payment/http-notification.md)
 </div>
 
-## Specify VA Number
+## Specifying VA Number
 
-Virtual Account number displayed to customer contains two parts. for example, in `{91012}{12435678}` , the first part is the company code and the second part is a unique code. The second part can be customized (For **Permata** only supported for b2b agreement model).
+Virtual Account number which is displayed to customer, contains two parts. for example, in `{91012}{12435678}` , the first part is the company code and the second part is a unique code. The second part can be customized (For **Permata**  b2b agreement model is supported). Following conditions need to be followed while customizing VA number:
```

Comment:
double space before the word b2b. And it's to be written as B2B. 
Last line to be rephrased.